src/views/TechnicalAssistance/RequestForm/Documents.tsx

```diff
@@ -1,29 +1,501 @@
+    <Switch>
+      {/* Documents table */}
+      <Route exact path="/trb/requests/:id/documents">
+        {/* Open the document upload form */}
+        <div className="margin-top-5 margin-bottom-4">
+          <UswdsReactLink
+            variant="unstyled"
+            className="usa-button"
+            to={`${url}/upload`}
+          >
+            {t('documents.addDocument')}
+          </UswdsReactLink>
+        </div>
+
+        <div>
+          <Table bordered={false} fullWidth scrollable {...getTableProps()}>
+            <thead>
+              {headerGroups.map(headerGroup => (
+                <tr {...headerGroup.getHeaderGroupProps()}>
+                  {headerGroup.headers.map((column, index) => (
+                    <th
+                      {...column.getHeaderProps(column.getSortByToggleProps())}
+                      aria-sort={getColumnSortStatus(column)}
+                      scope="col"
+                      className="border-bottom-2px"
+                    >
+                      <Button
+                        type="button"
+                        unstyled
+                        className="width-full display-flex"
+                        {...column.getSortByToggleProps()}
+                      >
+                        <div className="flex-fill text-no-wrap">
+                          {column.render('Header')}
+                        </div>
+                        <div className="position-relative width-205 margin-left-05">
+                          {getHeaderSortIcon(column)}
+                        </div>
+                      </Button>
+                    </th>
+                  ))}
+                </tr>
+              ))}
+            </thead>
+            <tbody {...getTableBodyProps()}>
+              {rows.map(row => {
+                prepareRow(row);
+                return (
+                  <tr {...row.getRowProps()}>
+                    {row.cells.map((cell, index) => {
+                      return (
+                        <td {...cell.getCellProps()}>{cell.render('Cell')}</td>
+                      );
+                    })}
+                  </tr>
+                );
+              })}
+            </tbody>
+          </Table>
+          {data && documents.length === 0 && (
+            <div className="font-body-2xs margin-left-2">
+              {t('documents.table.noDocument')}
+            </div>
+          )}
+        </div>
+
+        <Pager
+          className="margin-top-7"
+          back={{
+            onClick: () => {
+              history.push(stepUrl.back);
+            }
+          }}
+          next={{
+            onClick: e => {
+              history.push(stepUrl.next);
+            },
+            text: t(
+              documents.length
+                ? 'button.next'
+                : 'documents.continueWithoutAdding'
+            ),
+            outline: documents.length === 0
+          }}
+          taskListUrl={taskListUrl}
+        />
```

Comment:
I think this component is missing a `submit` prop/callback.  Clicking 'Save and exit" does nothing.